docs/contributing/tests.md

```diff
@@ -114,6 +114,11 @@ TEST_SAVE_LOGS=info pnpm --workspace-root test:blackbox
 
 The log files will be available under `tests/blackbox/server-logs-*`.
 
+### Run Blackbox Tests in Pull Requests
+
+Full blackbox tests run automatically when changes are merged into `main`. To trigger blackbox tests on a pull request,
+add the `Run Blackbox` label.
+
```

Comment:
@danielBreitlauch Good point! 👍 I have added this to the documentation. Does this addition suffice?

Reply:
Yes very true! Updated it to the Blackbox readme in https://github.com/directus/directus/pull/24517/commits/a36072a52e6c4caae4a9d659cfe85b0b18cabf80